css/index.css

```diff
@@ -1,3 +1,356 @@
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+:root{
+    font-size: 16px;
+}
+
+@media only screen and (max-width: 800px) {
+	:root{
+        font-size: 14px; 
+	}
+ }
+
+html, body {
+    height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+}
+
+@media only screen and (max-width: 800px) {
+	body{
+        line-height: 1.5; 
+	}
+ }
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body{
+	max-width: 1024px;
+	margin: 0 auto;
+	display: flex;
+	flex-direction: column;
+	font-size: 1em;
+}
+@media only screen and (max-width: 1024px) {
+	body{
+		margin: 2rem;
+	}
+ }
+
+h1{
+	font-size: 4rem;
+}
+
+h2{
+	font-size: 3rem;
+}
+
+h3{
+	font-size: 1.5rem;
+}
+@media only screen and (max-width: 800px) {
+    h2{
+        font-size: 2rem;
+    }
+    }
+
+@media only screen and (max-width: 800px) {
+     h3{
+        font-size: 1rem;
+    }
+}
+    
+span.divider{
+	flex-grow:1;
+}
+
+/* Navigation
+===========================================*/
+
+.navigation{
+	margin-top: 0.6rem;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;	
+}
+
+@media only screen and (max-width: 800px){
+    .navigation{
+        flex-direction: column;
+        margin-top: 0.3rem;
+    }
+
+    .logo{
+        padding: 1rem;
+    }
+}
+
+nav{
+	width: 72%;
+	display: flex;
+	flex-wrap: wrap;
+    justify-content: space-evenly;
+    padding: 1rem;
+}
+
+@media only screen and (max-width: 800px){
+    nav{
+        width: 100%;
+        order: 2;
+    } 
+}
+
+@media only screen and (max-width: 500px){
+    nav{
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    } 
+    
+    nav a{
+        padding: 1rem;
+        border-bottom: 1px solid black;
+        width: 100%;
+        text-align: center;
+    }
+    nav a:first-child{
+        border-top: 1px solid black;
+    }
+}
+
+a:link {
+	text-decoration: none;
+	color: lightgray;
+}
+a:visited{
+    text-decoration: none;
+    color: inherit;
+}
+a:hover {
+	text-decoration: none;
+	font-weight: bold;
+	color: dodgerblue;
+}
+
+
+
+/* Homepage */
+
+.header-container{
+	margin: 1.8rem 0;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+    align-items: center;
+    padding: 2rem 0;
+}
+
+header{
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	text-align: center;
+}
+
+
+header h1 {
+	flex-direction: column;
+	align-items: center;
+}
+
+@media only screen and (max-width: 800px) {
+	img.header-img{
+		display: none;
+	}
+}
+
+.get-started{
+	border: 1px solid black;
+	padding: 0.6rem 2rem;
+}
+
+.get-started a:link{
+	color: black;
+}
+
+.get-started:hover{
+	background-color: dodgerblue;
+}
```

Comment:
Again really like the effect. I would probably do the html like this: 
```
<a href="#">
          <button class="get-started">
            Get Started
          </button>
        </a>
```

and then set ```cursor: pointer;``` on the .get-started class. This will stop the weird effect of the text disappearing when hovered and the pointer only appearing when the text is hovered over.